runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeSearchPanel.java

```diff
@@ -222,10 +223,12 @@ private void processResult(List<ItemPrice> result, String lookup, boolean exactM
 		SwingUtilities.invokeLater(() ->
 		{
 			int index = 0;
+			int natRune = itemManager.getItemPrice(ItemID.NATURE_RUNE);
```

Comment:
Small nitpick, but this can be added directly to L231 without assignment to a variable since its value won't be reused.

Reply:
conflicts with @Adam review 15 days ago around looking it up once

Reply:
Good call, didn't notice the loop where GrandExchangeItemPanels are initialized. Adam's comment is definitely right to be followed here. 👍